### PR TITLE
skopeo: add page

### PR DESCRIPTION
**Pull request by gjbianco**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/skopeo.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# skopeo
+
+> Container image management toolbox.
+> Provides various utility commands to manage remote container images.
+> More information: <https://github.com/containers/skopeo>.
+
+- Inspect a remote image from a registry:
+
+`skopeo inspect docker://{{registry.hostname}}/{{image:tag}}`
```
Comment (navarroaxel) [inline pages/common/skopeo.md:9]: If this is a placeholder we use `snake_case` for them
```suggestion
`skopeo inspect docker://{{registry_hostname}}/{{image:tag}}`
```

File: pages/common/skopeo.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- List available tags for a remote image:
+
+`skopeo list-tags docker://{{registry.hostname}}/{{image}}`
```
Comment (navarroaxel) [inline pages/common/skopeo.md:13]:
```suggestion
`skopeo list-tags docker://{{registry_hostname}}/{{image}}`
```

File: pages/common/skopeo.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- Download an image from a registry:
+
+`skopeo copy docker://{{registry.hostname}}/{{image:tag}} dir:{{path/to/directory/}}`
```
Comment (navarroaxel) [inline pages/common/skopeo.md:17]: Is the `/` required?
```suggestion
`skopeo copy docker://{{registry_hostname}}/{{image:tag}} dir:{{path/to/directory}}`
```

File: pages/common/skopeo.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Copy an image from one registry to another:
+
+`skopeo copy docker://{source.registry}}/{{image:tag}} docker://{{destination.registry}}/{{image:tag}}`
```
Comment (navarroaxel) [inline pages/common/skopeo.md:21]: A typo here
```suggestion
`skopeo copy docker://{{source_registry}}/{{image:tag}} docker://{{destination_registry}}/{{image:tag}}`
```

File: pages/common/skopeo.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- Delete an image from a registry:
+
+`skopeo delete docker://{{registry.hostname}}/{{image:tag}}`
```
Comment (navarroaxel) [inline pages/common/skopeo.md:25]:
```suggestion
`skopeo delete docker://{{registry_hostname}}/{{image:tag}}`
```

File: pages/common/skopeo.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+- Log in to a registry:
+
+`skopeo login --username {{username}} {{registry.hostname}}`
```
Comment (navarroaxel) [inline pages/common/skopeo.md:29]:
```suggestion
`skopeo login --username {{username}} {{registry_hostname}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**